app/finders/conversation_finder.rb

```diff
@@ -23,6 +23,8 @@ def initialize(current_user, params)
   def perform
     set_up
 
+    filter_by_reply_status
```

Comment:
Why don't you add this filter inside `set_up` method?
```
filter_by_reply_status if params[:reply_status] == 'unattended'
```

Reply:
Done